conductor/server/config/config.properties

```diff
@@ -0,0 +1,36 @@
+# Database persistence model.  Possible values are memory, redis, and dynomite.
+# If ommitted, the persistence used is memory
+#
+# memory : The data is stored in memory and lost when the server dies.  Useful for testing or demo
+# redis : non-Dynomite based redis instance
+# dynomite : Dynomite cluster.  Use this for HA configuration.
+
+db=dynomite
+
+# Dynomite Cluster details.
+# format is host:port:rack separated by semicolon
+workflow.dynomite.cluster.hosts=mender-dynomite:8102:us-east-1c
```

Comment:
uhh, just changing to `dummyregion` doesn't work, as if the `us-east-1c` is the single blessed value.

I found this bit of comment however:
https://github.com/Netflix/conductor/blob/2ae02585087f8808d2da52188d854b503339031a/server/src/main/resources/server.properties#L39

after setting `EC2_AVAILABILITY_ZONE=dummyregion` it works. but I'm not sure it's not even more confusing, at some point we'll have to figure out what it means exactly.